sass/includes/_record-links.scss

```diff
@@ -0,0 +1,92 @@
+.record-links {
+    &__wrapper {
+        display: flex;
+        align-items: center;
+        flex-direction: column;
+        padding-left:10%;
+        padding-right:10%;
+
+        @media (max-width: $screen__md) {
+            padding-left:5%;
+            padding-right:5%;
+        }
+    }
+
+    &__list {
+        list-style: none;
+        margin-bottom: 0px;
+        width: 100%;
+        padding-left: 0;
+        border-top: 0.3rem solid #ffcc00;
+    }
+
+    &__list a {
+        text-decoration: none;
+        color: inherit;
+    }
+    
+    &__list a:hover &__item,
+    &__list a:focus &__item{
+        outline: 0.312rem solid $color__dark;
+    }
+
+    &__item {
+        display: flex;
+        align-items: center;
+        border: 1px solid #D9D9D6;
+        padding-top: 2rem;
+        padding-bottom: 2rem;
+        width: 100%;
+
+        @media (max-width: $screen__lg) {
+            display: block;
+        }
+
+        &__thumbnail {
+            flex-shrink: 0;
+            width: 7.5rem;
+            height: 7.5rem;
+            margin-left: 2rem;
+    
+            @media (max-width: $screen__lg) {
+                margin-bottom: 1rem;
+            }
+        }
+
+        &__thumbnail-image {
+            width: 100%;
+            height: 100%;
+            object-fit: cover;
+        }
+
+        &__text {
+            flex: 100%;
+            padding-left: 2rem;
+            padding-right: 2rem;
+        }
+
+        &__title {
+            margin: 5px 0;
+            color: #000000;
+            text-decoration-line: underline;
+            font-family: $font__roboto-mono;
+            font-style: normal;
+            font-weight: 700;
```

Comment:
We're importing the roboto font from google, but it looks like we're missing the bold version as the '700' weight isn't actually having any effect on the title. Let me know if you're not sure how to update this